tests/dummy/app/components/configurable-table.js

```diff
@@ -0,0 +1,13 @@
+import Ember from 'ember';
+import TableComponent from 'ember-table/components/ember-table';
+
+// TODO(azirbel): We extend this to create a very hacky way of calling
+// `@onResizeEnd` in the table, triggered by resizing the table's container. We
+// only need to do this because the table currently does a bad job detecting
+// when its width has changed.
+export default TableComponent.extend({
+  layoutName: 'components/ember-table',
+  parentWidthObserver: Ember.computed(function() {
```

Comment:
uhm humm. It is supposed to be an observer! Weird... https://github.com/Addepar/ember-table/blob/724cb261c45b87344d20c7d814a6622f3621ab44/app/controllers/configurable-columns.coffee#L131